src/pages/EventAdministration/components/EventFormAdmin.tsx

```diff
@@ -1,20 +1,100 @@
-import { EventFormCreate } from 'types';
+import { EventFormCreate, Form, FormCreate } from 'types';
 import { EventFormType, FormResourceType } from 'types/Enums';
+import { useState } from 'react';
 import { useEventById } from 'hooks/Event';
-import { useCreateForm } from 'hooks/Form';
+import { useForm } from 'react-hook-form';
 import { Typography, LinearProgress, Stack, Button } from '@mui/material';
 
 // Project
+import { useSnackbar } from 'hooks/Snackbar';
+import { useCreateForm, useFormTemplates, useDeleteForm, useFormById } from 'hooks/Form';
+import { removeIdsFromFields } from 'hooks/Utils';
+import VerifyDialog from 'components/layout/VerifyDialog';
+import FormView from 'components/forms/FormView';
+import Expand from 'components/layout/Expand';
 import FormAdmin from 'components/forms/FormAdmin';
 import { ShowMoreText } from 'components/miscellaneous/UserInformation';
+import FormFieldsEditor from 'components/forms/FormFieldsEditor';
 
 export type EventFormAdminProps = {
   eventId: number;
 };
 
+export type EventFormEditorProps = {
+  eventId: number;
+  formId: string | null;
+  formType: EventFormType;
+};
+
+type FormTemplatePreviewType = Omit<EventFormEditorProps, 'formId'> & {
+  formtemplate: Form;
+};
+
+const FormTemplatePreview = ({ formtemplate, eventId, formType }: FormTemplatePreviewType) => {
+  const deleteForm = useDeleteForm(formtemplate.id || '-');
+  const { register, formState, getValues, control } = useForm<Form['fields']>();
+  const [isEditing, setIsEditing] = useState(false);
+  const createForm = useCreateForm();
+  const showSnackbar = useSnackbar();
+
+  const onCreateFormFromTemplate = async () => {
+    const newForm: EventFormCreate = {
+      title: String(eventId),
+      type: formType,
+      event: eventId,
+      resource_type: FormResourceType.EVENT_FORM,
+      fields: removeIdsFromFields(formtemplate.fields),
+      template: false,
+    };
+    createForm.mutate(newForm, {
+      onSuccess: (data) => {
+        showSnackbar(data.title, 'success');
+      },
+      onError: (e) => {
+        showSnackbar(e.detail, 'error');
+      },
+    });
+  };
+
+  const onDeleteForm = () => {
+    deleteForm.mutate(undefined, {
+      onSuccess: (data) => {
+        showSnackbar(data.detail, 'success');
+      },
+      onError: (e) => {
+        showSnackbar(e.detail, 'error');
+      },
+    });
+  };
+
+  return (
+    <Expand flat header={formtemplate.title} sx={{ mt: 1 }}>
```

Comment:
Dropp marginTop for å unngå mellomrom mellom templates'ene:
<img width="541" alt="image" src="https://user-images.githubusercontent.com/31009729/151131036-c5aad25b-730b-4ada-965c-98729810bc06.png">

```suggestion
    <Expand flat header={formtemplate.title}>
```